internal/events/token_pool_created.go

```diff
@@ -53,6 +53,7 @@ func (em *eventManager) confirmPool(ctx context.Context, pool *fftypes.TokenPool
 	if err := em.persistBlockchainEvent(ctx, chainEvent); err != nil {
 		return err
 	}
+	em.emitBlockchainEventMetric(*ev)
```

Comment:
Just wondering why the mem copy here, rather than a pass-by-reference?
I think the blockchain.Event structure is reasonably large.